### PR TITLE
Add: styling and responsiveness

### PR DESCRIPTION
**Pull request by mrlobatoman**
## Describe your changes:
[//]: <> (Which files were changed? What changes were made?)
Image.css, header.css, app.css - some styling adjustments and media queries.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have tested the functionality of any new or updated features in the browser, if applicable
- [x] I have added a "Reviewer" to review & merge
- [] If applicable, my code passes all tests

If issue found, describe and refernces user story/projectboard issue

### CODE REVIEW

**Comment by mrlobatoman** [comment]
Desktop:
Screenshot 2023-05-27 at 12 00 34 PM

Ipad:
Screenshot 2023-05-27 at 12 38 31 PM

Cellphone:
Screenshot 2023-05-27 at 12 38 04 PM

File: src/components/header/Header.css [hunk 1]
```diff
@@ -1,14 +1,26 @@
 header {
   display: flex;
   justify-content: space-between;
+  padding-top: 1%;
```
Comment (sdtrachsel) [inline src/components/header/Header.css:4]: We should be able to DRY this up the padding using css short hand.

**Review by sdtrachsel — COMMENTED** [review]
Starting to look like a real website! We will need to make a correction on mobile to keep the Saved Memes link from wrapping. I also noted where we could dry up the styling using css short hand. 

Nice work!

**Review by sdtrachsel — APPROVED** [review]
Starting to look like a real website! We will need to make a correction on mobile to keep the Saved Memes link from wrapping. I also noted where we could dry up the styling using css short hand. 

Nice work!

**Review by CodeMeister362 — APPROVED** [review]

**Pull request merged**